### ISSUE TITLE
Exception on flow expiry with "use_idle_timeout: True"

### ISSUE THREAD

**Issue by simeonmiteff** [faucetsdn/faucet#1153]
When setting the dp configuration `use_idle_timeout: True`, upon flow expiry, I get:

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/faucet/valve_util.py", line 34, in __koe
    func(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/faucet/faucet.py", line 531, in flowremoved_handler
    flowmods = valve.flow_timeout(msg.table_id, msg.match)
  File "/usr/lib/python3.6/site-packages/faucet/valve.py", line 909, in flow_timeout
    return self.host_manager(table_id, match)
TypeError: 'ValveHostFlowRemovedManager' object is not callable
```
In valve.py:

https://github.com/faucetsdn/faucet/blob/394329e517c3e21770a7c6a42cd49883c6c5252f/faucet/valve.py#L909

I believe it should read:
```python
        return self.host_manager.flow_timeout(table_id, match)
```

**Comment by simeonmiteff** [comment]
@gizmoguy is quick: https://github.com/gizmoguy/faucet/commit/32c647a7e4d900598636ece5fca28fa9623aab37

**Comment by gizmoguy** [comment]
Fixed in #1154 

Thanks for the report!

**Comment by anarkiwi** [comment]
I don't recommend using this feature. It is known to be unreliable (flow
removed message is not reliable and there may be other flaws such as
dependency on order of flow removed messages). It is so noted in the unit
tests.
